eureka-protempa-etl/src/main/java/edu/emory/cci/aiw/cvrg/eureka/etl/resource/JobResource.java

```diff
@@ -126,17 +126,27 @@ public JobResource(JobDao inJobDao, TaskManager inTaskManager,
 	@GET
 	@Transactional
 	public List<Job> getAll(@Context HttpServletRequest request,
-			@QueryParam("order") String order) {
-		JobFilter jobFilter = new JobFilter(null,
-				this.authenticationSupport.getUser(request).getId(), null, null, null);
+							@QueryParam("order") String order,
+							@QueryParam("recent") boolean recent) {
```

Comment:
getAll() is trying to do too many things. Make "recent" a separate endpoint, like /jobs/latest

Use "latest" instead of "recent".

Also, make it return just one record.